cpp/src/parquet/arrow/arrow_reader_writer_test.cc

```diff
@@ -1342,6 +1360,31 @@ TEST_F(TestUInt32ParquetIO, Parquet_1_0_Compatibility) {
 
 using TestStringParquetIO = TestParquetIO<::arrow::StringType>;
 
+TEST_F(TestStringParquetIO, Basics) {
+  std::shared_ptr<Array> values;
+
+  ::arrow::StringBuilder builder;
+  for (size_t i = 0; i < SMALL_SIZE; i++) {
+    ASSERT_OK(builder.Append("abc"));
+  }
+  ASSERT_OK(builder.Finish(&values));
+
+  // Input is narrow array, but expected output is large array, opposite of the above tests.
```

Comment:
The comment seems to be copied from line 1437? `above tests` looks confusing here.